### PR TITLE
fix(console): change "color-text-inverse-weak" for a11y

### PR DESCRIPTION
**Pull request by serifluous**
- [x] In console theme, `color-text-inverse-weak` changed to `palette-gray-50` to pass AA contrast with `color-background-inverse`

It didn't originally pass AA as its original value of `palette-gray-60`.

### CODE REVIEW

File: packages/paste-design-tokens/tokens/global/text-color.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@ props:
     value: "{!palette-gray-0}"
     comment: Inverse text color for dark backgrounds
   color-text-inverse-weak:
-    value: "{!palette-gray-60}"
+    value: "{!palette-gray-50}"
```
Comment (TheSisb) [inline packages/paste-design-tokens/tokens/global/text-color.yml]: Is the -60 value still fine on the Sendgrid theme?

File: packages/paste-design-tokens/tokens/global/text-color.yml [hunk 2]
```diff
@@ -16,7 +16,7 @@ props:
...
     comment: Weak inverse text color for dark backgrounds
```
Comment (TheSisb) [inline packages/paste-design-tokens/tokens/global/text-color.yml]: Are there any notes or gotchas related to this change we should add in this comment line?

File: packages/paste-design-tokens/tokens/global/text-color.yml (hunk 1 above)
Comment (serifluous) [reply packages/paste-design-tokens/tokens/global/text-color.yml]: Yep, it passes AA (https://www.aremycoloursaccessible.com/?background=%2308152c&colorCombos=%5Bobject%20Object%5D&colorCombos=%5Bobject%20Object%5D&isLight=false&textColor=%239ca7ba)

File: packages/paste-design-tokens/tokens/global/text-color.yml (hunk 2 above)
Comment (serifluous) [reply packages/paste-design-tokens/tokens/global/text-color.yml]: Do you mean whether we should change the comment to, e.g., "Weak inverse text color for dark backgrounds. Must pass AA color contrast with color-background-inverse"?

Or did you have something else in mind?

Comment (TheSisb) [reply packages/paste-design-tokens/tokens/global/text-color.yml]: Nothing specific in mind, just wanted to make sure to ask.  What you wrote above seems good to me!

File: packages/paste-design-tokens/tokens/global/text-color.yml (hunk 1 above)
Comment (serifluous) [reply packages/paste-design-tokens/tokens/global/text-color.yml]: ok actually, just noticed `color-background-inverse` isn't defined in the SendGrid theme tokens. I'll add it.

**Pull request merged**